composer.json

```diff
@@ -5,118 +5,115 @@
+        "chamilo/pclzip": "~2.8",
+        "clue/graph": "~0.9.0",
+        "culqi/culqi-php": "1.3.4",
+        "ddeboer/data-import": "@stable",
+        "doctrine/data-fixtures": "~1.0@dev",
         "doctrine/dbal": "~2.4",
         "doctrine/migrations": "~1.0@dev",
-        "symfony/console": "~2.3",
-        "symfony/yaml": "~2.3",
-        "symfony/doctrine-bridge": "~2.8",
-        "symfony/validator": "~2.8",
-        "symfony/filesystem": "~2.8",
-        "symfony/http-foundation": "~2.8",
-        "symfony/security": "~2.8",
-        "symfony/serializer": "~3.0",
-        "doctrine/data-fixtures": "~1.0@dev",
-        "knplabs/gaufrette": "~0.3",
-        "friendsofsymfony/user-bundle": "~1.3",
-        "sonata-project/admin-bundle": "~3.1",
-        "sonata-project/core-bundle": "~3.1",
-        "sonata-project/user-bundle": "3.3.0",
-        "toin0u/digitalocean": "1.4.2",
-        "michelf/php-markdown": "~1.7",
-        "sabre/vobject": "~3.1",
-
+        "doctrine/orm": "~2.4",
         "emojione/emojione": "1.3.0",
-
-        "zendframework/zend-config": "~2.6|^3.0",
-        "zendframework/zend-feed": "~2.6|^3.0",
-        "zendframework/zend-http": "~2.6|^3.0",
-        "zendframework/zend-soap": "~2.6|^3.0",
-        "robrichards/xmlseclibs": "3.0.*",
-
+        "essence/essence": "2.6.1",
+        "ext-curl": "*",
+        "ext-dom": "*",
+        "ext-fileinfo": "*",
+        "ext-gd": "*",
+        "ext-intl": "*",
+        "ext-json": "*",
+        "ext-libxml": "*",
+        "ext-mbstring": "*",
+        "ext-xml": "*",
+        "ext-zip": "*",
+        "ext-zlib": "*",
         "ezyang/htmlpurifier": "4.9.*",
-        "szymach/c-pchart": "3.0.6",
-        "aferrandini/phpqrcode": "1.0.1",
-        "mpdf/mpdf": "6.1.*",
-        "studio-42/elfinder": "2.1.*",
-        "jbroadway/urlify": "1.1.0-stable",
-        "monolog/monolog": "~1.0",
-        "ircmaxell/password-compat": "~1.0.4",
-        "sylius/attribute": "0.13.0",
-        "sylius/translation": "0.13.0",
-        "patchwork/utf8": "~1.2",
-        "ddeboer/data-import": "@stable",
-        "league/csv": "~8.0",
-        "phpoffice/phpexcel": "~1.8",
-        "webit/eval-math": "1.0.1",
-        "clue/graph": "~0.9.0",
-        "graphp/graphviz": "~0.2.0",
+        "facebook/php-sdk-v4": "~5.0",
+        "friendsofsymfony/user-bundle": "~1.3",
         "graphp/algorithms": "~0.8.0",
-        "sunra/php-simple-html-dom-parser": "~1.5.0",
-        "media-alchemyst/media-alchemyst": "~0.5",
-        "php-ffmpeg/php-ffmpeg": "0.5.1",
+        "graphp/graphviz": "~0.2.0",
+        "guzzlehttp/guzzle": "~6.0",
         "imagine/imagine": "0.6.3",
+        "ircmaxell/password-compat": "~1.0.4",
+        "jbroadway/urlify": "1.1.0-stable",
         "jeroendesloovere/vcard": "~1.7",
-        "knplabs/doctrine-behaviors": "~1.1",
         "jimmiw/php-time-ago": "0.4.15",
-        "phpoffice/phpword": "~0.14",
-        "facebook/php-sdk-v4": "~5.0",
```

Comment:
@marvil07 This one is missing in the new version. I know it says it's deprecated, but removing it in a sorting commit is not a good idea (and yes, I'm checking those one by one because it's very important)

Reply:
@ywarnier Please see my comment, I pointed where in the diff it is. I may be missing something here.

Reply:
I meant https://github.com/chamilo/chamilo-lms/pull/3023#discussion_r345497297